src/tests/data/gov_notify_body.json

```diff
@@ -0,0 +1,4 @@
+{
+    "template_id":"d4cbcc38-3d95-4e53-9614-5b0356e3407a",
+    "email_address":"dan.page@digital.cabinet-office.gov.uk"
```

Comment:
If a valid email is needed can we pull from .env variables or if a valid email is not needed replace with hardcoded invalid email address

Reply:
Thanks, updated to test@domain.com